pkg/bpfdebug/debug.go

```diff
@@ -12,93 +12,104 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package bpf
+package bpfdebug
 
 import (
 	"fmt"
 
 	"github.com/cilium/cilium/common"
 )
 
+// Must be synchronized with <bpf/lib/common.h>
```

Comment:
I want to get away from cgo long term. It seriously slows down the build and requires a ton of memory. These numbers are absolutely set in stone and can never change again.